### PR TITLE
[174] Calculate Product Shipping Cost - Fullstack

### PR DESCRIPTION
**Pull request by kojusola**
## Task: `created a file to calculate the product price and shipping cost`
- `Calculated the product shipping cost, total cost and product price`

### CODE REVIEW

File: submissions/adeolaFafemi/dom/productCost.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
+// selecting elements
+var productPrice = document.getElementById('productprice');
+var shippingCost= document.getElementById('shippingcost');
+var totalCost = document.getElementById('totalcost');
+var input = document.getElementById('quantity');
+var errorInput = document.getElementById('error-input');
+var buyNow = document.getElementById('submit')
+var alphaExp = /^[0-9]+$/;
```
Comment (adaugochi) [inline submissions/adeolaFafemi/dom/productCost.js:8]: This variable name is confusing. `alpha` sounds like it is alphabetical expression

File: submissions/adeolaFafemi/dom/productCost.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    } else{
+        errorInput.innerText = "";
+        buyNow.disabled = false;
+    }
+    updatePrice()
+    updateShipping()
+    updateTotal()
+}
+function updatePrice(){
+    var total = 0;
```
Comment (adaugochi) [inline submissions/adeolaFafemi/dom/productCost.js:26]: I don't think there is a need for this variable

**Review by adaugochi — CHANGES_REQUESTED** [review]
Well done. Kindly format your code properly. closing curly brackets should be properly placed. The opening curly bracket should be after the closing bracket `) {`

**Comment by kojusola** [comment]
made the corrections 
thank you

File: submissions/adeolaFafemi/dom/productCost.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    updateTotal()
+}
+function updatePrice(){
+        var quantity = input.value;
+        var total = (price*quantity); 
+        productPrice.innerText = total;
+    }
+function updateShipping(){
+    var prodprice = productPrice.innerText;
+    if (prodprice < 1000){
```
Comment (adaugochi) [inline submissions/adeolaFafemi/dom/productCost.js:32]: This block code is confusing. i.e line `32 - 43`. Are you wrapping the `else if` statements inside the `if` block? If yes, that is wrong. If no, kindly format your code to look well align

**Review by adaugochi — CHANGES_REQUESTED** [review]

**Comment by kojusola** [comment]
already reformatted it.
Thank you

**Review by adaugochi — APPROVED** [review]

**Pull request merged**